env/terragrunt.hcl

```diff
@@ -1,14 +1,15 @@
 locals {
-  vars = read_terragrunt_config("../env_vars.hcl")
+  vars        = read_terragrunt_config("../env_vars.hcl")
+  secret_vars = try(read_terragrunt_config("../secret_env_vars.hcl"), { inputs = {} })
```

Comment:
I'm concerned that if someone manages to inject a secret_env_vars.hcl file into our repo that we could potentially have a security incident, can you add an alarm that will trigger an incident if a secret_env_vars.hcl file is in this repo? And also something that will prevent the Pipeline from applying if the file is in the filesystem

Reply:
This also might be overkill as if someone did that then we are already compromised and they would have the ability to remove the alarms and modify the workflow file.

Reply:
That's fair - I'll leave it as-is for now.